src/static/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
Ensure its UTF-8 encoded

Reply:
Is the action item to explicitly serve a content encoding header with this file? Or is there something else we need to do?

Reply:
Just to be sure when creating the txt file, should be UTF-8 encoded. In the answer header we should have : content-type: text/plain; charset=utf-8

Reply:
Ok yeah Flask returns a header with this charset.

> `Content-Type: text/plain; charset=utf-8`